src/org/usfirst/frc/team467/robot/Autonomous/ActionGroup.java

```diff
@@ -158,7 +154,8 @@ public boolean isDone() {
 			} else {
 				increment = 0;
 			}
-			if (increment >= 5) {
+			// Each iteration is 20 ms
+			if (increment >= (RobotMap.AUTONOMOUS_DRIVE_TIMEOUT_MS / 20)) {
```

Comment:
This check for not moving for some number of iterations should have comments describing how it works.